src/styles/blocks/catalog.scss

```
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: 300ms;
+
```

Its bad practice to not write property which you want to be animated 
```suggestion
    transition: color 300ms, transform 300ms;

```

---

> Its bad practice to not write property which you want to be animated

thx, changed